src/main/resources/tosca-definitions/normative-types.yml

```diff
@@ -335,14 +335,14 @@ node_types:
             attached to specific Compute node instance
           capability: tosca.capabilities.network.Bindable
           relationship: tosca.relationships.network.BindsTo
-          node: tosca.nodes.Compute
+#          node: tosca.nodes.Compute
```

Comment:
Why are you changing the normative definitions? are they not aligned with the specification?

Reply:
Sorry, pasted in from Marica or Miguel... Don't remember reason (changed 14/02) Maybe to avoid some malfunctioning

Reply:
If you don't remember the reason, I'd put it in another PR then